lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3208,50 +3208,13 @@ namespace {
       // @defs is an anachronism; ignore it.
       return nullptr;
     }
+    
+    Decl *VisitVarDecl(const clang::VarDecl *decl) {
 
-    AccessorDecl *tryCreateConstexprAccessor(const clang::VarDecl *clangVar,
-                                             VarDecl *swiftVar) {
-      assert(clangVar->isConstexpr());
-      clangVar->evaluateValue();
-      auto evaluated = clangVar->getEvaluatedValue();
-      if (!evaluated)
-        return nullptr;
-
-      // If we have a constexpr var with an evaluated value, try to create an
-      // accessor for it. If we can remove all references to the global (which
-      // we should be able to do for constexprs) then we can remove the global
-      // entirely.
-      auto accessor = AccessorDecl::create(
-          Impl.SwiftContext, SourceLoc(), SourceLoc(), AccessorKind::Get,
-          swiftVar, SourceLoc(), StaticSpellingKind::KeywordStatic,
-          /*Async=*/false, /*AsyncLoc=*/SourceLoc(),
-          /*throws=*/false, SourceLoc(), /*genericParams*/ nullptr,
-          ParameterList::createEmpty(Impl.SwiftContext), swiftVar->getType(),
-          swiftVar->getDeclContext());
-      Expr *value = nullptr;
-      // TODO: add non-numeric types.
-      if (evaluated->isInt()) {
-        value = IntegerLiteralExpr::createFromUnsigned(
-            Impl.SwiftContext,
-            static_cast<unsigned>(
-                clangVar->getEvaluatedValue()->getInt().getZExtValue()));
-      } else if (evaluated->isFloat()) {
-        auto floatStr = evaluated->getAsString(Impl.getClangASTContext(),
-                                               clangVar->getType());
-        auto floatIdent = Impl.SwiftContext.getIdentifier(floatStr);
-        value = new (Impl.SwiftContext)
-            FloatLiteralExpr(floatIdent.str(), SourceLoc());
-      }
-      if (!value)
+      // For now we don't import constexpr
+      if (decl->isConstexpr())
```

Comment:
This should only be static constexpr vars, right? Other vars should work fine (it's just because there's no symbol for static constexpr vars).